mlflow/server/js/src/experiment-tracking/components/modals/BulkDeleteExperimentModal.tsx

```diff
@@ -0,0 +1,57 @@
+import { Typography } from '@databricks/design-system';
+import { ExperimentEntity } from '../../types';
+import { ConfirmModal } from './ConfirmModal';
+import { deleteExperimentApi } from '../../actions';
+import { useDispatch } from 'react-redux';
+import { ThunkDispatch } from '@mlflow/mlflow/src/redux-types';
+import Utils from '@mlflow/mlflow/src/common/utils/Utils';
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  experiments: Pick<ExperimentEntity, 'experimentId' | 'name'>[];
+  onExperimentsDeleted: () => void;
+};
+
+export const BulkDeleteExperimentModal = ({ isOpen, onClose, experiments, onExperimentsDeleted }: Props) => {
+  const dispatch = useDispatch<ThunkDispatch>();
+
+  const handleSubmit = () => {
+    return Promise.all(experiments.map((experiment) => dispatch(deleteExperimentApi(experiment.experimentId))))
+      .then(onExperimentsDeleted)
+      .catch((e: any) => Utils.logErrorAndNotifyUser(e));
+  };
+
+  return (
+    <ConfirmModal
+      isOpen={isOpen}
+      onClose={onClose}
+      handleSubmit={handleSubmit}
+      title={`Delete ${experiments.length} Experiment(s)`}
+      helpText={
+        <div>
+          <Typography.Paragraph>The following experiments will be deleted:</Typography.Paragraph>
+          <Typography.Paragraph>
+            <ul>
+              {experiments.map((experiment) => (
+                <li key={experiment.experimentId}>
+                  <Typography.Text>
+                    Experiment "{experiment.name}" (Experiment ID: {experiment.experimentId})
```

Comment:
i think we can simplify here:
```suggestion
                    {experiment.name} (ID: {experiment.experimentId})
```

Reply:
My previous comment applies to this as well, wanted to keep the messaging similar, but it may get repetitive so I'm applying your change.